recipes/pyod/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyod" %}
+{% set version = "0.7.7.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22cd28595187ca54e06418522b699b80ab6eae658ac3de194c3ef85730127016
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - combo
+    - joblib
+    - matplotlib
+    - numba >=0.35
+    - numpy >=1.13
+    - pip
+    - python
+    - scikit_learn >=0.19.1
+    - scipy >=0.19.1
+    - six
+    - suod
+  run:
+    - combo
+    - joblib
+    - matplotlib
+    - numba >=0.35
+    - numpy >=1.13
+    - python
+    - scikit_learn >=0.19.1
+    - scipy >=0.19.1
+    - six
+    - suod
+
+test:
+  imports:
+    - pyod
+    - pyod.models
+    - pyod.utils
+
+about:
+  home: "https://github.com/yzhao062/pyod"
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: "A Python Toolkit for Scalable Outlier Detection (Anomaly Detection)"
+  doc_url: "https://pyod.readthedocs.io"
+
+extra:
+  recipe-maintainers:
+    - yzhao062
```

Comment:
@yzhao062 are you willing to be a maintainer?

Reply:
Hi @scopatz! @yzhao062 agreed to be a maintainer here: https://github.com/yzhao062/pyod/issues/164

I should have put this in the PR. I'm sorry

Reply:
@yzhao062  will have to agree here